projects/common/src/telemetry/track/track.directive.ts

```diff
@@ -48,12 +54,12 @@ export class TrackDirective implements OnChanges, OnDestroy {
   }
 
   private clearListeners(): void {
-    this.subscription.unsubscribe();
+    this.activeSubscriptions.unsubscribe();
   }
 
-  private trackUserEvent(userEvent: TrackUserEventsType, eventObj: MouseEvent): void {
+  private trackUserEvent(userEvent: string, eventObj: MouseEvent): void {
     this.userTelemetryImplService.trackEvent(`${userEvent}: ${this.trackedEventLabel}`, {
-      target: eventObj.target,
+      ...(eventObj.target as HTMLElement),
```

Comment:
why are you spreading the whole element onto the event?

Reply:
I wasn't sure what all fields would be interesting. Anyway, I have added few properties only now.